.travis/kat-check.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+###
+# Checks that all KATs pass.
+###
+
+PRINT_GREEN="tput setaf 2"
```

Comment:
I suggest to import .travis/defs.sh instead of duplicating code.

Reply:
Done.